cohttp/test/test_header.ml

```diff
@@ -438,11 +431,6 @@ let link_ext_star () =
       ]
     (H.get_links headers)
 
-let trim_ws () =
```

Comment:
Why not keeping this test and running it instead?

Reply:
Done. Added to the test suite.